RatingSlider/Sources/RatingSlider.swift

```diff
@@ -22,40 +30,50 @@ public class RatingSlider: UIControl {
             updatedSize()
         }
     }
-    @IBInspectable public var font: UIFont {
-        get { return activeGrid.font }
-        set { grids { $0.font = newValue } }
-    }
     
     // MARK: Active grid
     
-    @IBInspectable public var activeLabelsColor: UIColor {
-        get { return activeGrid.textColor }
-        set { activeGrid.textColor = newValue }
+    @IBInspectable public var activeColor: UIColor {
+        get { return activeGrid.itemColor }
+        set { activeGrid.itemColor = newValue }
     }
+    
     @IBInspectable public var activeTrackColor: UIColor? {
         get { return activeGrid.backgroundColor }
         set { activeGrid.backgroundColor = newValue }
     }
+    
     public override var tintColor: UIColor! {
         didSet { activeTrackColor = tintColor }
     }
     
     // MARK: Inactive grid
     
-    @IBInspectable public var inactiveLabelsColor: UIColor {
-        get { return inactiveGrid.textColor }
-        set { inactiveGrid.textColor = newValue }
+    @IBInspectable public var inactiveColor: UIColor {
+        get { return inactiveGrid.itemColor }
+        set { inactiveGrid.itemColor = newValue }
     }
+    
     @IBInspectable public var inactiveTrackColor: UIColor? {
-        get { return backgroundColor }
-        set { backgroundColor = newValue }
+        get { return inactiveGrid.backgroundColor }
+        set { inactiveGrid.backgroundColor = newValue }
     }
     
     // MARK: - Init
     
-    public override init(frame: CGRect) {
+    public init(frame: CGRect,
+                gridStyle: GridStyle = .labels(font: .systemFont(ofSize: 12)),
+                gridHeight: CGFloat? = nil,
+                hasUpperGrid: Bool = false,
```

Comment:
It feels that these three are all related 😄 What would it mean to have defined `gridHeight` but style `.labels`? Does `hasUpperGrid` make sense together with `.labels`?

Reply:
Yep... Upper Grid shouldn't came together with `.labels(...)` style. 🙅‍♂️
So `gridHeight` and `hasUpperGrid` should be in `.dots()` style.

Reply:
Should we add these parameters to `GridStyle.dots` case then?

Reply:
Yes